src/Core/src/Hosting/Internal/AppHostEnvironment.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.Maui.Hosting.Internal
 {
-	internal class AppHostEnvironment : IHostEnvironment
+	class AppHostEnvironment : IHostEnvironment
```

Comment:
Why this change? if not internal maybe move it out of this namespace/folder

Reply:
This is just removing the keyword as internal is default.